apps/ledger-live-desktop/src/renderer/components/TranslatedError.jsx

```diff
@@ -43,7 +47,21 @@ class TranslatedError extends PureComponent<Props> {
             typeof str === "string" ? <li key={key}>{str}</li> : null,
           );
         }
-        return translation;
+
+        const learnMoreClickable = urls.errors[error.name] ? (
+          <Text ff="Inter|SemiBold">
+            <ExternalLink
+              label={t("common.learnMore")}
+              onClick={() => openURL(urls.errors[error.name])}
+            />
+          </Text>
+        ) : null;
+
+        return (
+          <Text>
```

Comment:
Being extra careful here, but do you think `<Text>` can never change the display, depending on the context? I'm just wondering because of how `TranslatedComponent` is used in so many places...

Reply:
I'll try to do an extra condition for not impacting unless we got a learn more